test/testtoken.cpp

```diff
@@ -653,7 +653,7 @@ class TestToken : public TestFixture {
         givenACodeSampleToTokenize floatingPoint("0.0f", true);
         ASSERT_EQUALS(true, Token::Match(floatingPoint.tokens(), "%num%"));
 
-        givenACodeSampleToTokenize doublePrecision("0.0d", true);
+        givenACodeSampleToTokenize doublePrecision("0.0", true);
```

Comment:
I do not know where 0.0d comes from. I don't want to immediately say if we need that or not.

Reply:
It seems to come from this commit: acad87c3e34dbb5fd7005fd637eea1cf807f5a92

And I don't see why that was added it could be by mistake. I guess we can remove `0.0d`.